src/App/components/Navbar/Navbar.styles.js

```diff
@@ -1,71 +1,254 @@
 import styled from "styled-components";
-import { Row, Icon } from "antd";
+import { Row, Input, Avatar } from "antd";
 
 export const NavContainer = styled(Row)`
-  background-color: #444;
+  background-color: #3e4257;
   width: 100%;
-  padding: 5px 0px 5px 200px;
+  height: 4.375rem;
+  display: flex;
   align-items: center;
+  font-family: Roboto;
 `;
+
 export const LogoContainer = styled.div`
-  display: flex;
-  align-items: center;
+  padding-left: 5px;
 `;
-export const LogoImg = styled.img`
-  width: 45px;
-  height: 45px;
-  margin-right: 10px;
+export const CurrentPage = styled.h3`
+  display: none;
+
+  @media (min-width: 1000px) {
+    display: block;
+    min-width: 12%;
+    font-size: 0.875rem;
+    margin: 0 0 0 1rem;
+    text-align: center;
+    color: #ffffff;
+    font-weight: bold;
+  }
 `;
-export const LogoHeader = styled.h2`
-  color: white;
-  line-height: 1;
-  margin-bottom: 0;
+
+export const AppLogo = styled.img`
+  width: 2.5rem;
+  height: 2.5rem;
+
+  @media (min-width: 1000px) {
```

Comment:
Can we follow the mobile-first approach? You can swap these values and make it max-width of `1000px` I believe. Also, while making use of constants, please define the constants above.

Reply:
I'm already using mobile-first approach? The min-width is for larger screens. The default styles are for mobile. Could you tell me what do you mean exactly?